main/lsp/requests/rename.cc

```diff
@@ -228,6 +228,19 @@ class MethodRenamer : public Renamer {
         if (location == nullptr) {
             return;
         }
+
+        // Have we already edited this exact location?
```

Comment:
Rather than eagerly dedupe, should we dedupe at the end? I think that'd be cheaper. Regardless, I'm OK with approving this as-is so we can get this in.

Reply:
Hmm you're right, n^2 vs lg n in the number of edits in one file. Or maybe we should load it into a set and turn it into a vector at the end. I think I'll do this is in a separate change so we can at least unblock this use case for now.

Reply:
Sorbet's `loc` data structure is much smaller to store and cheaper to compare than the JSON data structures (3 numbers rather than string + numbers), so we could consider lazily converting edits into the desired JSON-style data structure.